tests/null-test/main.tf

```
@@ -0,0 +1,7 @@
+module "null_s3account" {
+  source = "../.."
+
+  s3_account = {}
```

This is testing an empty map as an input, not `null`.

```suggestion
  s3_account = null
```